mlflow/models/evaluation/base.py

```diff
@@ -739,9 +737,7 @@ def evaluate(
                 the evaluator.
             model: A pyfunc model instance. If None, the model output is supposed to be found in
                 ``dataset.predictions_data``.
-            custom_metrics: Deprecated. Use ``extra_metrics`` instead.
             extra_metrics: A list of :py:class:`EvaluationMetric` objects.
-            custom_artifacts: A list of callable custom artifact functions.
```

Comment:
Do we also need to remove `custom_artifacts`?

Reply:
good catch. my mistake. reverted the change.